CHANGELOG.md

```diff
@@ -50,6 +50,8 @@ messages inside `authz.MsgExec`
 
 * [#113](https://github.com/babylonlabs-io/babylon/pull/113) Add multibuild binary
 for upgrade handler `testnet` and `mainnet`.
+* [#140](https://github.com/babylonlabs-io/babylon/pull/140) Removed `unbonding`
```

Comment:
this should probably be a bug fix

Reply:
moved at https://github.com/babylonlabs-io/babylon/pull/140/commits/d851cb1f0b5acaccea7182a9acf68f942568376b